shared/profile/wot-author/index.tsx

```diff
@@ -170,12 +192,7 @@ export const Question1 = (props: Question1Props) => {
   const onSubmit = () => {
     props.onSubmit({
       otherText: selectedVerificationType === 'other' ? otherText : '',
-      proofs: proofs
-        .filter(({checked}) => checked)
-        .map(proof => {
-          const {checked, onCheck, ...rest} = proof
-          return rest
-        }),
+      proofs: proofs.filter(({checked}) => checked).map(proof => proof.wotProof),
```

Comment:
NAB: i just don't know javascript well enough, but i'd be surprised if there aren't more terse ways to write this.

Reply:
Use `reduce`!